solution_code/js/index.js

```diff
@@ -0,0 +1,30 @@
+
+/* Show and hide readmore */
+
+$(document).ready(function(){   /* make a function available after the document is loaded */
+
+document.getElementById("readmore").addEventListener("click", function(event){
+    event.preventDefault()
+    $('.readmore').hide();
+    $('.readless').show();
+    $('#show-this-on-click').slideDown();
+});
+
+
+document.getElementById("readless").addEventListener("click", function(event){
+    event.preventDefault()
+    $('.readless').hide();
+    $('.readmore').show();
+    $('#show-this-on-click').slideUp();
```

Comment:
You can add a function in slideUp/SlideDown for actions you want to execute when the sliding action has finished

```
$('#show-this-on-click').slideUp(function(){
    $('.readless').hide();
    $('.readmore').show();
});
```